src/Abstract/Pokemon.java

```diff
@@ -0,0 +1,53 @@
+package Abstract;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class Pokemon {
+    private String name;
+    private List<String> types;
+
+    //constructor
+    public Pokemon(String name, String type) {
+        this.name = name;
+        this.types = new ArrayList<String>();
+        this.types.add(type);
+    }
+
+    public Pokemon(String name, List<String> types) {
+            this.name = name;
+            this.types = types;
+    }
+
+    //methods
+    public void addType(String type) {
+        this.types.add(type);
+    }
+    public void describe() {
+        System.out.println(name + " is a " + types + "-type Pokémon.");
```

Comment:
public void describe() {
            String alltypes = "";
            for (int i = 0; i < types.size(); i++) {
                if(i == types.size() - 1){
                    alltypes += types.get(i) + " ";
                }
                else{
                    alltypes += types.get(i) + ", ";
                }
            }
            System.out.println(name + " is a " + alltypes + "type Pokémon.");
            
    }
    
    
    Als je dit van je describe had gemaakt dan had je alle types netjes gehad buiten de brackets :)

Reply:
We hadden het natuurlijk al over de aanvallen gehad :)